server/mergin/auth/models.py

```diff
@@ -35,6 +35,10 @@ class User(db.Model):
     __table_args__ = (
         db.Index("ix_user_username", func.lower(username), unique=True),
         db.Index("ix_user_email", func.lower(email), unique=True),
+        CheckConstraint(
```

Comment:
This will fail on checking so-called anonymized email with pattern "deleted_\d{13}". Can we exclude it? 
Else we would need to update our deleting strategy and migrate db.